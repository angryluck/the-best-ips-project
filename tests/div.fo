fun int div() =
  let n = read(int) in
  let d = read(int) in
  write(n/d)

fun int main() =
  let tmp = div() in
  let tmp = div() in
  let tmp = div() in
  let tmp = div() in
  let tmp = div() in
  let tmp = div() in
  let tmp = div() in
  div()
