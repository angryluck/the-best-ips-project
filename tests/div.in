8
2
1
8
0
16
-0
13
-12
7
84584
-100
-123
-12
