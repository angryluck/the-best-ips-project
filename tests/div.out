4
0
0
0
-1
-845
10
